lib/elasticsearch_wrapper/synonyms.rb

```diff
@@ -0,0 +1,45 @@
+module ElasticsearchWrapper
+  class Synonyms
+    SYNONYM_SET_NAME = "dynamic_synonyms_#{Rails.env}"
+    def self.client
+      @@client ||= Elasticsearch::Client.new
```

Comment:
This is not going to connect to the prod server correctly -- can we use `Elasticsearch::Model.client` which is set [here](https://github.com/ApprenticeshipStandardsDotOrg/ApprenticeshipStandardsDotOrg/blob/main/config/initializers/elasticsearch.rb#L15)?